runtime/onert/core/include/ir/train/TrainingInfo.h

```diff
@@ -44,12 +50,17 @@ class TrainingInfo final
   const OptimizerInfo &optimizerInfo() const { return _optimizer_info; }
   uint32_t batchSize() const { return _batch_size; }
   const uint32_t &trainingStep() const { return _training_step; }
+  const std::unordered_set<OperationIndex> &getTrainableOps() const { return _trainable_ops; }
```

Comment:
I tend to avoid `std::unordered_set` and use `std::set` instead, because the former adds non-determinism to the code. (in particular, it randomizes the iteration order)

Is there a reason to use it over `std::set`?

Reply:
Wouldn't it be okay to use `std::unordered_set`if the iteration order is not important and hash values are guaranteed?

Reply:
> @Aeren1564 Wouldn't it be okay to use `std::unordered_set`if the iteration order is not important and hash values are guaranteed?

Then the future contributors will have to be very careful not to iterate it in a context where the iteration order has visible side effects, or we could alternatively allow non-deterministic behavior.

Reply:
OK I see. It would be better to use `std::set` here to prevent side effects by trying iteration of it as you mentioned.
By the way, do you think it's OK with using `std::unordered_set` when finding or inserting an element in places where performance(latency) is important when finding  (not iteration)? There are places in `onert` where performance is important.

Reply:
Yes I think it's ok to use it for performance as long as it's not being iterated.

Reply:
Thanks for you opinion. I agree with you.

Reply:
@jyoungyun Could you change it to `std::set` as @Aeren1564 mentioned?